frontend/common/crud/useGet.tsx

```diff
@@ -11,8 +11,10 @@ export function useGet<T>(
   url: string | undefined,
   query?: Record<string, string | number | boolean>,
   swrConfiguration: SWRConfiguration = {}
+  // disables running the request, useful when you dont want to actualy run the hook in certain conditions
```

Comment:
what does this comment belongs with?

if it's about swrConfiguration, put it above,
if about useGet, put it inside

Reply:
Oh, thanks for catching that! I tried to implemenent conditional request, but it showed that useSWR actualy can do this by passing empty URL.